docs/articles/documentation/reference/test-api/testcontroller/fixturectx.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+title: t.fixtureCtx Property
+permalink: /documentation/reference/test-api/testcontroller/fixturectx.html
+---
+# t.fixtureCtx Property
+
+Allows you to access the *fixture context* from test code. Use it to share variables between [fixture hook functions](../../../guides/basic-guides/test-organization.md#fixture-hooks) and test code.
+
+```text
+t.fixtureCtx
+```
+
+Fixture hooks take the `ctx` parameter that allows you to access the fixture context.
+
+```js
+fixture `Fixture1`
+    .before(async ctx  => {
+        ctx.someProp = 123;
+    })
+    .after(async ctx  => {
+        console.log(ctx.someProp); // > 123
+    });
+```
+
+Test code can read the fixture context from `t.fixtureCtx`, assign to its properties or add new ones, but it cannot overwrite the entire `t.fixtureCtx` object.
```

Comment:
assign what to its properties?

Reply:
*Test code can read the fixture context from `t.fixtureCtx`, assign values to its properties or add new properties, but it cannot overwrite the entire `t.fixtureCtx` object.*

Reply:
ok good